### PR TITLE
Improve documentation and quickstart

### LINKED ISSUE #5: Improve quickstart and docs (now that there is a docker image available)

### PR DESCRIPTION
**Pull request by supernarthur**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,32 +4,68 @@ Export license usage for SideFX license server to Prometheus.
...
 - `/opt/hfs<version>/houdini/sbin/sesictrl`
 
-The version of houdini (which translates 1:1 with the version of these binaries)
-should be above the version of your license server. At the time of writing, the
-exporter has been tested with versions 19.5 and 20.
+Make sure your Houdini version matches or is newer than your license server
+version. Currently, this exporter has been tested with versions 19.5 and 20.
+
+The available Docker compose configuration expects these binaries to be
+installed in the `./hfs-bin/` directory, but you can use the directory of your
```
Comment (doubleailes) [inline README.md]: Can you precise the way to choose the directory?

Comment (bobafouette) [reply README.md]: There might be a bit of complexity between the docker-compose and the `config.yml` file that will be interpreted from inside the container that could be worth explaining :+1:

**Comment by bobafouette** [comment]
I wonder if it is for this MR or we should have a dedicated one but it could be interesting to mention in the documentation that this can also be installed as a python package. It's a cool feature that could interest the non docker crowd.

What do you think ?

**Comment by supernarthur** [comment]
> I wonder if it is for this MR or we should have a dedicated one but it could be interesting to mention in the documentation that this can also be installed as a python package. It's a cool feature that could interest the non docker crowd.
> 
> What do you think ?

I feel like it's fairly unlikely to be used like this, knowing the general recommended use case of prometheus exporters. And so any way to make the documentation a bit leaner is an added bonus. 
I guess that if there is some interest we can jump on it and make pypi packages in the CI.

File: README.md (hunk 1 above)
Comment (supernarthur) [reply README.md]: I fixed that in 84eb035, hopefully this is better.

**Review by bobafouette — APPROVED** [review]

**Pull request merged**